andela-ifatoki/CP2-Shift-DMS
# #148921725 Create new documents

#### What does this PR do?
It implements a feature in which a signed in user can create new documents
#### Description of Task to be completed?
A Signed in user should be able to create new documents which reflect on his home page in realtime
#### How should this be manually tested?
- Sign in with valid details and click on the **Create New Document** icon on the top right hand part of the screen. 
- You would get a popup where you can type in your document details and title.
- Click on the **save** icon to your bottom right hand side. 
- You would see the new document appear among your listed documents on the home page.
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
148921725
#### Screenshots (if appropriate)
<img width="1232" alt="screen shot 2017-07-15 at 10 25 56 am" src="https://user-images.githubusercontent.com/28437656/28238311-d1fe0648-6948-11e7-9695-f85eabb4c14e.png">
<img width="1215" alt="screen shot 2017-07-15 at 10 26 26 am" src="https://user-images.githubusercontent.com/28437656/28238312-d22ca2fa-6948-11e7-8cc8-de34405bb700.png">
<img width="1130" alt="screen shot 2017-07-15 at 10 27 12 am" src="https://user-images.githubusercontent.com/28437656/28238313-d2318c84-6948-11e7-82e0-5e1e0d01fc08.png">
<img width="1203" alt="screen shot 2017-07-15 at 10 27 21 am" src="https://user-images.githubusercontent.com/28437656/28238314-d2618d3a-6948-11e7-81e6-ff6a56564fc7.png">
#### Questions:
N/A
